### PR TITLE
SampleGen: Conform to Ruby style requirements

### LINKED ISSUE #2812: SampleGen: update Ruby snippets to match official googleapis/ruby-style
Style and accompanying lint configuration file: [googleapis/ruby-style](https://github.com/googleapis/ruby-style)
Run lint the code, follow instructions in README of [googleapis/ruby-style](https://github.com/googleapis/ruby-style)

As reported by: https://github.com/googleapis/google-cloud-ruby/pull/3431#issuecomment-498772695

/cc @googleapis/samplegen 
/fyi @TheRoyalTnetennba @dazuma @quartzmo

### PR DESCRIPTION
**Pull request by hzyi-google**
Fix #2812

### CODE REVIEW

File: src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline [hunk 1]
```diff
@@ -5776,12 +5783,10 @@ require "library"
...
- # Testing resource name overlap
- #
- # @param shelf {String} Test word wrapping for long lines. This is a long comment. The name of the
- # shelf to retrieve the big book from.
- # @param big_book_name {String} The name of the book.
-def sample_get_big_book(shelf, big_book_name)
+
+
+
+def sample_get_big_book shelf, big_book_name
```
Comment (beccasaurus) [inline src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline]: Shouldn't this have parens now?

File: src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline [hunk 2]
```diff
@@ -6090,11 +6101,9 @@ require "library"
 
 # [START canonical]
 
- # Testing <@calling forms>
- #
- # @param shelf_name {String} The name of the shelf where books are published to.
```
Comment (beccasaurus) [inline src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline]: Where'd the `@param`s go?

File: src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline (hunk 1 above)
Comment (hzyi-google) [reply src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline]: I thought rubocop does not like parens?

Copying from rubocop report:

```
samples/v1/speech_transcribe_multichannel.rb:24:21: C: Style/MethodDefParentheses: Use def without parentheses.
def sample_recognize(local_file_path)
                    ^^^^^^^^^^^^^^^^^
```

Or does that only apply to the case where there is only one parameters?

File: src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline (hunk 2 above)
Comment (hzyi-google) [reply src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline]: Derp. They are back now.

File: src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline (hunk 1 above)
Comment (beccasaurus) [reply src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline]: Ohhhh, we _had_ them before, I didn't realize we do.

Ok, +1

I couldn't remember what ruby-style's option was of this. Nvm, LGTM :D

**Pull request merged**